cores/arduino/FspTimer.cpp

```diff
@@ -470,6 +476,37 @@ bool FspTimer::set_period(uint32_t p) {
 
 
 
+/* -------------------------------------------------------------------------- */
+bool FspTimer::use_period_buffer(bool buffer_period) {
```

Comment:
I think this should be called  `set_period_buffer` just to match other functions.

```suggestion
bool FspTimer::use_period_buffer(bool buffer_period) {
```